.github/workflows/ci.yml

```diff
@@ -111,10 +111,8 @@ jobs:
         with:
           save-if: ${{ github.event_name != 'merge_group' }}
 
-      # TODO: Add --target x86_64-unknown-none to the no_std check once we solve the compilation issues with it.
-      #       https://github.com/linebender/parley/issues/86
```

Comment:
What's the status of https://github.com/linebender/parley/issues/86 once this PR is merged? I don't think this is currently marked as fixing that issue.

Reply:
Needs to be reviewed!